opossum-tool/OpossumUI
# feat: use short name of common licenses as license name if selected

### Summary of changes

If the user selects a common license as license of the attribution, the short name (SPDX identifier) is stored as license name of the attribution.

### Context and reason for change

fixes issue #2588

### How can the changes be tested

See the new e2e test: Update an attribution by selecting one of the common licenses and note that the short name (the name in brackets in the list of common licenses) is set as license name for the attribution.
